third_party/maya/lib/usdMaya/xformStack.h

```diff
@@ -0,0 +1,315 @@
+
+    PXRUSDMAYA_API
+    size_t GetSize() const;
+
+
+    /// \brief  Finds the index of the Op Classification with the given name in this stack
+    /// \param  opName the name of the operator classification we  wish to find
+    /// \param  isInvertedTwin the returned op classification object must match
+    ///         this param for it's IsInvertedTwin() - if an op is found that matches
+    ///         the name, but has the wrong invertedTwin status, nullptr is returned
+    /// return  Undex to the op classification object with the given name (and
+    ///         inverted twin state); will be NO_INDEX if no match could be found.
+    PXRUSDMAYA_API
+    size_t FindOpIndex(const TfToken& opName, bool isInvertedTwin=false) const;
+
+    /// \brief  Finds the Op Classification with the given name in this stack
+    /// \param  opName the name of the operator classification we  wish to find
+    /// \param  isInvertedTwin the returned op classification object must match
+    ///         this param for it's IsInvertedTwin() - if an op is found that matches
+    ///         the name, but has the wrong invertedTwin status, nullptr is returned
+    /// return  Pointer to the op classification object with the given name (and
+    ///         inverted twin state); will be nullptr if no match could be found.
+    PXRUSDMAYA_API
+    const OpClass& FindOp(const TfToken& opName, bool isInvertedTwin=false) const;
+
+    /// \brief  Finds the indices of Op Classification(s) with the given name in this stack
+    /// \param  opName the name of the operator classification we  wish to find
+    /// return  A pair of indices to op classification objects with the given name;
+    ///         if the objects are part of an inverted twin pair, then both are
+    ///         returned (in the order they appear in this stack). If found, but
+    ///         not as part of an inverted twin pair, the first result will point
+    ///         to the found classification, and the second will be NO_INDEX.  If
+    ///         no matches are found, both will be NO_INDEX.
+    PXRUSDMAYA_API
+    const IndexPair& FindOpIndexPair(const TfToken& opName) const;
+
+    /// \brief  Finds the Op Classification(s) with the given name in this stack
+    /// \param  opName the name of the operator classification we  wish to find
+    /// return  A pair of pointers to op classification objects with the given name;
+    ///         if the objects are part of an inverted twin pair, then both are
+    ///         returned (in the order they appear in this stack). If found, but
+    ///         not as part of an inverted twin pair, the first result will point
+    ///         to the found classification, and the second will be null.  If
+    ///         no matches are found, both pointers will be nullptr.
+    PXRUSDMAYA_API
+    const OpClassPair FindOpPair(const TfToken& opName) const;
+
+    /// \brief Returns a list of pointers to matching XformOpDefinitions for this stack
```

Comment:
Doc for returned value needs to be updated, since it's no longer a list of pointers